tests/DataFactories/WooCommerceProduct.php

```diff
@@ -22,7 +22,9 @@ function __construct(\AcceptanceTester $tester) {
       'name' => 'Product',
       'type' => self::TYPE_SIMPLE,
       'sku' => 'WC_PR_'. uniqid(),
-      'price' => 10
+      'price' => 10,
+      'categoryId' => null,
```

Comment:
This way we can assign the product only to one category or tag. I think it would be better to make it `categoryIds` and then the related method would be`withCategoryIds` and store ids as array of arrays:
```php
$ids = array_map(function($id){ return ['id' => $id]; }, $ids);
```
That way when you put categories into command you can do just: `'--categories=' . json_encode($this->data['categoryIds])`

Same for tags